coffee/lib/abstract-chosen.coffee

```diff
@@ -280,8 +281,19 @@ class AbstractChosen
     if /Android/i.test(window.navigator.userAgent)
       return false if /Mobile/i.test(window.navigator.userAgent)
     return true
+    
+  @tagname_is_valid: : (@form_field) ->
+  if @form_field.tagName.toUpperCase() != 'SELECT'
+    console.error 'Chosen error: invalid tagName ' + @form_field + ' is not a select'
+    return false
+
+  if @form_field.getElementsByTagName('option').length == 0
+    console.error 'Chosen error: ' + @form_field + ' has no options\'s childs '
```

Comment:
You can use [CoffeeScript's existential operator](http://coffeescript.org/#operators-existential) here instead:

```coffeescript
console?.error "..."
```